### ISSUE TITLE
Error with numpy?

### ISSUE THREAD

**Issue by denics** [Ayima/sitemap-visualization-tool#9]
Hi I have reinstalled sitemap-visualization-tool recently on a new debian based machine and I am getting this error:
```
$ python categorize_urls.py --depth 4
ImportError: No module named _multiarray_umath
```

That I think refers to numpy library. This is the installed version:
```
Requirement already satisfied: numpy in /usr/local/lib/python2.7/dist-packages (1.15.4)
```

Denis

**Comment by zazencodes** [comment]
Hi Denis, have you tried upgrading numpy?
```
pip install --upgrade numpy
```

**Comment by denics** [comment]
Hi agalea91, this should be fixed by #10 ;)
